.goreleaser.yml

```diff
@@ -0,0 +1,33 @@
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
```

Comment:
```suggestion
    ldflags:
        - -X github.com/pulumi/crd2pulumi/gen.Version={{ .Version }}
    goos:
```

Reply:
we actually don't need ldflags for this repo - there is no version being accepted as part of the file - maybe I am missing something here?

Reply:
The previous Makefile had something like:
```make
VERSION := 1.0.5
LDFLAGS := "-X 'github.com/pulumi/crd2pulumi/gen.Version=$(VERSION)'"
GOOS=darwin GOARCH=amd64 $(GO) build -ldflags=$(LDFLAGS) -o releases/crd2pulumi-darwin-amd64/crd2pulumi $(PROJECT)
```
so that the `gen.Version` variable was set. Although `gen.Version` didn't affect any core logic, just the output of `crd2pulumi --version`.

I'm not familiar with GoReleaser, but I think we do need to be able to accept some `{{ .Version }}` as a part of the file. They did something similar [here](https://goreleaser.com/customization/build/), but I'm where/how they got the `{{ .Version }}` from, or whether it's something that's implicitly set with every build.